components/host-dashboard/reports-section/TransactionsOverviewSection.js

```diff
@@ -0,0 +1,258 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useQuery } from '@apollo/client';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+
+import { formatCurrency } from '../../../lib/currency-utils';
+import { API_V2_CONTEXT, gqlV2 } from '../../../lib/graphql/helpers';
+
+import PeriodFilter from '../../budget/filters/PeriodFilter';
+import CollectivePickerAsync from '../../CollectivePickerAsync';
+import Container from '../../Container';
+import { Flex } from '../../Grid';
+import Loading from '../../Loading';
+import { P, Span } from '../../Text';
+
+const FilterLabel = styled.label`
+  font-weight: 500;
+  text-transform: uppercase;
+  margin-bottom: 8px;
+  color: #4e5052;
+`;
+
+const FundAmounts = styled.div`
+  width: 100%;
+  height: 48px;
+  border-radius: 10px;
+  background-color: #29cc75;
+  border-right: 360px solid #e03f6a;
+  padding-top: 10px;
+  padding-left: 5px;
+  @media (max-width: 1025px) {
+    height: 130px;
+    background-color: #29cc75;
+    border-right: 0px;
+    border-bottom: 40px solid #e03f6a;
+  }
+`;
+
+const TotalFundsLabel = styled(Container)`
+  display: table-cell;
+  padding-left: 10px;
+  height: 26px;
+  border-radius: 5px;
+  background-color: white;
+  opacity: 80%;
+  vertical-align: middle;
+`;
+
+const Square = styled(Container)`
+  width: 8px;
+  height: 8px;
+  display: inline-block;
+  background-color: ${props => props.color};
+`;
+
+const transactionsOverviewQuery = gqlV2/* GraphQL */ `
+  query TransactionsOverviewQuery(
+    $hostSlug: String!
+    $accounts: [AccountReferenceInput!]
+    $dateFrom: String
+    $dateTo: String
+  ) {
+    host(slug: $hostSlug) {
+      id
+      contributionStats(accounts: $accounts, from: $dateFrom, to: $dateTo) {
+        numContributions
+        numOneTime
+        numRecurring
+        dailyAvgIncome
+      }
+      expenseStats(accounts: $accounts, from: $dateFrom, to: $dateTo) {
+        numExpenses
+        dailyAverage
+        numInvoices
+        numReimbursements
+        numGrants
+      }
+    }
+  }
+`;
+
+const TransactionsOverviewSection = ({ hostSlug, currency }) => {
```

Comment:
:ice_cream: Since we're fetching data anyway, I would add `currency` to the query and retrieve it from `data.host.currency`